Update indicator-manage.md (#10183)
MicrosoftDocs/microsoft-365-docs: microsoft-365/security/defender-endpoint/indicator-manage.md

5. Select **Done**.

> [!NOTE]
> Only 500 indicators can be uploaded for each batch. 

> 
> > [!NOTE]
> > Attempting to import Indicators with specific categories requires the string to be written on Pascal Case convention and only accepts the category list available at the portal.

Reviewer: ```suggestion
>
> Attempting to import indicators with specific categories requires the string to be written in Pascal case convention and only accepts the category list available at the portal.
```
